specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -57,8 +133,14 @@ model Evaluation {
   @visibility(Lifecycle.Read)
   name: string;
 
-  @doc("Data for evaluation.")
-  data: InputData;
+  @doc("Deprecated: data for evaluation.")
+  data?: InputData;
```

Comment:
How will the case where customer provides
- Data and model 

In this case evaluation API does inference and then evaluation (current behaviour)